### DISCUSSION TITLE
Will communication distance be affected by lora mode

### DISCUSSION THREAD

**Discussion by weekroom** [jgromes/RadioLib#1208] [Q&A]
At present, I have two sx1262 lora devices in my hand. When I set one to receive only and one to send only, the maximum communication distance can reach 1.2km with 915Mhz,bw=62.5,sf=12,CR=8. However, when I set one to send before receiving, and one to receive before sending, The communication distance between the two devices is only 800 meters, and I am working at 915Mhz,bw=62.5,sf=11,CR=8 (when sf=12, in the case of NFC, I receive a CRC error).

**Comment by HeadBoffin** [comment]
Yes, different LoRa settings will vary the range but there is no magic formula for the settings & range. However the position of the antenna's, the type and the terrain will have a huge impact on range. LoRa is in the frequency range where line of sight is important. If you are using SF12 and only getting 1.2km then you need to look at what is in the between the two antennas.

**Reply by weekroom** [reply]
The two test environments are the same, but the mode and spread spectrum factor are modified. Although the spread spectrum factor 1 is reduced, the distance may become shorter, but the actual packet loss rate is significantly increased. The odd thing is that SF can be set to 12 with only one mode, but not when the receiving and sending modes are switched back and forth, and I also gave enough time for the lora modem to modulate or demodulate

**Reply by HeadBoffin** [reply]
So very hard to comment without code and knowing where these devices are being run to look at the terrain, which is why you got a very generic answer. But it's usually about the antennas and the terrain, unless the radios have been abused or are faulty - the former by running them without an antenna, the latter incredibly rare.

My long distance record in free air with a metrological balloon was ~115km - piece of straight wire on the transmitter, ground station had a mid-range mag mount antenna - so the lack of obstructions helped.

Recent LoRaWAN mapping exercise by @StevenCellist in the dark reaches of Northern England reached 92km from the side of a hill to a gateway at sea level. This would not have been at a TTN / legal friendly SF, not going all out.
